src/Reflection/ReflectionMethod.php

```diff
@@ -23,22 +24,25 @@ class ReflectionMethod extends ReflectionFunctionAbstract
     /**
      * @param Reflector $reflector
      * @param MethodNode $node Node has to be processed by the PhpParser\NodeVisitor\NameResolver
+     * @param Namespace_|null $namespace
      * @param ReflectionClass $declaringClass
      * @param ReflectionClass $implementingClass
      */
     public static function createFromNode(
         Reflector $reflector,
         MethodNode $node,
+        ?Namespace_ $namespace,
         ReflectionClass $declaringClass,
         ReflectionClass $implementingClass
     ) : self {
         $method                    = new self();
         $method->declaringClass    = $declaringClass;
         $method->implementingClass = $implementingClass;
 
+        // @TODO I don't understand the following comment:
```

Comment:
Not sure what the following commit means, but the fact that a namespace is set or not should be exposed properly by the child class API.

I think this `TODO` as well as the comment should go away